content/posts/10-year-anniversary.md

```diff
@@ -0,0 +1,65 @@
+* Keeping trust in the community is the most important thing, because we're nothing without our users. It's a cliche, but trust is hard to earn and easy to lose. We continually think about how to balance sustainability with our the values of our users, and have made many decisions over the years about having "enough". **We can't maximize profit and continue to keep the trust of our community.**
+* **Contribution is easier for less complex parts of the code base.** We worked for a long time to get contributors to the core [readthedocs.org repo](https://github.com/readthedocs/readthedocs.org/), but the complexity of the application makes it really hard to contribute. We've had more success with contributions on our Sphinx theme, Sphinx extensions, and other small Python & Javascript projects.
+* **Being open source means capturing a small percentage of the value you create.** Back before we had a sustainable business model, I would get upset about the number of people telling me about how valuable it was for them hosting it themselves inside their company, while not contributing anything back. Now that we have a sustainable model and focused on having "enough" to keep the project going, I can see the benefits of the code being used in many places, even if we don't get direct contributions from those users. Though I do still wish they'd contribute back :)
+
+## Gratitude
+
+We are grateful to have a team of 4 folks working full-time on Read the Docs.
+The support of the Python community, the Django community, and the broader open source community has been very important to keeping us going over the years.
+We depend on the trust of the projects that use our platform, and we have a core value not to violate that trust.
+We're also thankful to be mostly bootstrapped, outside of 6% equity we gave to our initial incubator, so that we don't have to follow the *enshittification* path of many venture-backed companies.
+
+We are also grateful for the sponsored services for our Community site from many tech companies, most importantly [AWS](https://aws.amazon.com/) and [Cloudflare](https://www.cloudflare.com/).
+This allows us to focus our funding on the team and core code base, rather than infrastructure costs.
+
+## Next steps
+
+Read the Docs started only supporting Sphinx, and now supports any documentation tool that generates HTML output.
+We've also finished up a large "magic removal" branch that has removed our modification of the documentation build process,
+reducing the confusion that users hit building locally versus on Read the Docs.
+On top of that,
+we're layering our [Addons system](https://docs.readthedocs.io/en/stable/addons.html),
+which providers features for documentation readers on top of any tool.
```

Comment:
I think that's what this is saying?

Reply:
I meant this could discuss our vision a bit more broadly. Right now it reads more like we're describing a recent sprint -- ie the magic removal branch is nice but doesn't really feel like a vision for RTD.

Reply:
I mean, changing the build system to not modify user configs is a huge change, and is definitely part of the new vision for the product where we aren't changing user HTML at all.